src/guides/v2.4/graphql/mutations/create-customer-v2.md

```diff
@@ -0,0 +1,101 @@
+---
+group: graphql
+title: createCustomerV2 mutation
+---
+
+The `createCustomerV2` mutation creates a customer account. Use the [`createCustomerAddress` mutation]({{page.baseurl}}/graphql/mutations/create-customer-address.html) to complete the customer profile and define billing and shipping addresses.
+
+The `createCustomerV2` mutation replaces the `createCustomer` mutation as the means to create a customer account. The input objects differ between these two mutations. The `createCustomer` mutation required the `CustomerInput` object, as did the `updateCustomer` mutation. The attributes required for creating a customer are different than those for updating a customer, but you could not determine this by looking at the schema. The `createCustomerV2` mutation requires the `CustomerCreateInput` object, which it does not share with the [`updateCustomerV2` mutation]({{page.baseurl}}/graphql/mutations/update-customer-v2.html).
+
+## Syntax
+
+`mutation: {createCustomerV2(input: CustomerCreateInput!) {CustomerOutput}}`
+
+## Example usage
+
+The following call creates a new customer.
+
+**Request:**
+
+```graphql
+mutation {
+  createCustomerV2(
+    input: {
+      firstname: "Bob"
+      lastname: "Loblaw"
+      email: "bobloblaw@example.com"
+      password: "b0bl0bl@w"
+      is_subscribed: true
+    }
+  ) {
+    customer {
+      firstname
```

Comment:
did you leave on purpose all possible customer fields?
I guess yes because it's an example
maybe add a comment in the example # client can add other optional fields in the query
you can add comments wtih # in any graphql query

Reply:
They're just examples, not meant to show every possibility. Keeping the examples to typical and/or minimal use cases also helps maintain these examples, as I don't have to update the example each time an attribute is added.